### ISSUE TITLE
Program file (e.g. hello.jl) currently needs to be in static-julia directory

### ISSUE THREAD

**Issue by tkoolen** [JuliaComputing/static-julia#2]
Otherwise you get
```
twan@tk-loco:~/code/static-julia$ julia juliac.jl ~/code/RigidBodyDynamics/v0.6/RigidBodyDynamics/hello.jl /opt/julia-0.6/ $JULIA_PKGDIR 
Program File : /home/twan/code/RigidBodyDynamics/v0.6/RigidBodyDynamics/hello.jl
Julia Install Path: /opt/julia-0.6/
/usr/bin/ld: cannot open output file lib/home/twan/code/RigidBodyDynamics/v0.so: No such file or directory
collect2: error: ld returned 1 exit status
ERROR: LoadError: failed process: Process(`g++ -m64 -fPIC -shared -o lib/home/twan/code/RigidBodyDynamics/v0.so /home/twan/code/RigidBodyDynamics/v0.o -L/opt/julia-0.6/lib -Wl,--export-dynamic -Wl,-rpath,/opt/julia-0.6/lib -Wl,-rpath,/opt/julia-0.6/lib/julia -ljulia`, ProcessExited(1)) [1]
Stacktrace:
 [1] pipeline_error(::Base.Process) at ./process.jl:682
 [2] run(::Cmd) at ./process.jl:651
 [3] compile(::String, ::String, ::String) at /home/twan/code/static-julia/juliac.jl:34
 [4] include_from_node1(::String) at ./loading.jl:569
 [5] include(::String) at ./sysimg.jl:14
 [6] process_options(::Base.JLOptions) at ./client.jl:305
 [7] _start() at ./client.jl:371
while loading /home/twan/code/static-julia/juliac.jl, in expression starting on line 51
```

**Comment by tkoolen** [comment]
Note that the issue is still valid; the readme has just been updated.

**Comment by ViralBShah** [comment]
Good point. Maybe we should leave the issue open.

**Comment by ViralBShah** [comment]
@lucatrv Perhaps we can close this one too?

**Comment by lucatrv** [comment]
Yes also this one was fixed with PR #15
